src/peft/tuners/ia3/model.py

```diff
@@ -39,6 +39,11 @@
 
     from .bnb import Linear8bitLt
 
+if is_bnb_4bit_available():
+    import bitsandbytes as bnb
```

Comment:
Add the import statement `from .bnb import Linear8bitLt` in the conditional block for consistency.